metals/src/main/scala/scala/meta/internal/metals/codelenses/RunTestCodeLens.scala

```diff
@@ -57,25 +57,33 @@ final class RunTestCodeLens(
   ): Seq[l.CodeLens] = {
     val textDocument = textDocumentWithPath.textDocument
     val path = textDocumentWithPath.filePath
-    if (path.isAmmoniteScript || path.isWorksheet) {
-      Seq.empty
-    } else {
-      val distance = buffers.tokenEditDistance(path, textDocument.text, trees)
-      val lenses = for {
-        buildTargetId <- buildTargets.inverseSources(path)
-        buildTarget <- buildTargets.info(buildTargetId)
-        connection <- buildTargets.buildServerOf(buildTargetId)
-        // although hasDebug is already available in BSP capabilities
-        // see https://github.com/build-server-protocol/build-server-protocol/pull/161
-        // most of the bsp servers such as bloop and sbt might not support it.
-        if buildTarget.getCapabilities.getCanDebug || connection.isBloop || connection.isSbt,
-      } yield {
-        val classes = buildTargetClasses.classesOf(buildTargetId)
-        codeLenses(textDocument, buildTargetId, classes, distance, path)
-      }
+    val distance = buffers.tokenEditDistance(path, textDocument.text, trees)
+    val lenses = for {
+      buildTargetId <- buildTargets.inverseSources(path)
+      buildTarget <- buildTargets.info(buildTargetId)
+      connection <- buildTargets.buildServerOf(buildTargetId)
+      // although hasDebug is already available in BSP capabilities
+      // see https://github.com/build-server-protocol/build-server-protocol/pull/161
+      // most of the bsp servers such as bloop and sbt might not support it.
+    } yield {
+      val classes = buildTargetClasses.classesOf(buildTargetId)
+      if (connection.isScalaCLI && path.isAmmoniteScript) {
```

Comment:
`connection.isScalaCLI` but `path.isAmmoniteScript`? Simple alias for `isAmmoniteScript` like `def isScalaCliScript: Boolean = isAmmoniteScript` will be more readable